pages/stands/index.tsx

```diff
@@ -0,0 +1,165 @@
+import React from 'react';
+import { graphql } from 'react-relay';
+import { withDataAndLayout, WithDataAndLayoutProps } from '../../lib/withData';
+import PageView from '../../components/PageView';
+import styled from 'styled-components';
+
+import { stands_QueryResponse } from '../../__generated__/stands_Query.graphql';
+import { useEffect, useState } from 'react';
+import StandCard from '../../components/Stands/StandCard';
+import { currentDayCompanies, timeIsAfterNow } from '../../utils/time';
+import LivePlayer from '../../components/Stands/LivePlayer';
+
+// Update the currentEvent-list every 30 sec
+const intervalLength = 1000 * 30;
+
+const Index = ({
+  error,
+  props,
+}: WithDataAndLayoutProps<stands_QueryResponse>): JSX.Element => {
+  const [time, setTime] = useState(Date.now());
```

Comment:
If you use `dayjs()` instead of `Date.now()` you can get stronger types than `number` that you get with `Date`. Not strictly necessary, but it's nice to be consistent with the date objects in the app and types for the win!